Examples/SlackClone/SlackMessageOptionsResolver.swift

```diff
@@ -14,7 +14,9 @@ final class SlackMessageOptionsResolver: ChatMessageLayoutOptionsResolver {
         appearance: Appearance
     ) -> ChatMessageLayoutOptions {
         var options = super.optionsForMessage(at: indexPath, in: channel, with: messages, appearance: appearance)
-        options.remove([.flipped, .bubble, .timestamp, .avatar, .avatarSizePadding, .authorName, .threadInfo, .reactions])
+
+        [.flipped, .bubble, .timestamp, .avatar, .avatarSizePadding, .authorName, .threadInfo, .reactions]
+            .forEach { options.remove($0) }
```

Comment:
Why this change? Keep in mind, if you are getting a Warning, this was a mistake and it is fixed on 4.6.0 branch. Please revert this change

Reply:
I was getting a warning about that

Reply:
✅ Discarded this change